CMakeLists.txt

```diff
@@ -484,6 +484,22 @@ else()
   unset(c_flags_debug_overridden)
 endif()
 
+set(CMAKE_CXX_FLAGS_COVERAGE "-Og --coverage -fprofile-update=prefer-atomic")
```

Comment:
> fprofile-update=prefer-atomic

Not sure this should be in here? Don't think it's needed with Clang, and only sometimes with GCC?

Reply:
Would it hurt if applied in such cases? Or do you want me to remove it?

Reply:
> Not sure this should be in here?

Deleted.

The PR description has been updated.